.github/workflows/lint.yml

```diff
@@ -2,7 +2,7 @@ name: lint
 
 on:
   push:
-    branches: [ $default-branch ]
+    branches: [ main ]
```

Comment:
Correcting a silly oversight on my part from #11; `$default-branch` only works in templates, not actual workflow files.